src/main/cli/settings.c

```diff
@@ -914,6 +914,7 @@ const clivalue_t valueTable[] = {
     { "roll_rate_limit",            VAR_UINT16 | PROFILE_RATE_VALUE, .config.minmaxUnsigned = { CONTROL_RATE_CONFIG_RATE_LIMIT_MIN, CONTROL_RATE_CONFIG_RATE_LIMIT_MAX }, PG_CONTROL_RATE_PROFILES, offsetof(controlRateConfig_t, rate_limit[FD_ROLL]) },
     { "pitch_rate_limit",           VAR_UINT16 | PROFILE_RATE_VALUE, .config.minmaxUnsigned = { CONTROL_RATE_CONFIG_RATE_LIMIT_MIN, CONTROL_RATE_CONFIG_RATE_LIMIT_MAX }, PG_CONTROL_RATE_PROFILES, offsetof(controlRateConfig_t, rate_limit[FD_PITCH]) },
     { "yaw_rate_limit",             VAR_UINT16 | PROFILE_RATE_VALUE, .config.minmaxUnsigned = { CONTROL_RATE_CONFIG_RATE_LIMIT_MIN, CONTROL_RATE_CONFIG_RATE_LIMIT_MAX }, PG_CONTROL_RATE_PROFILES, offsetof(controlRateConfig_t, rate_limit[FD_YAW]) },
+    { "throttle_vbat_limit_percent", VAR_UINT8  | PROFILE_RATE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_CONTROL_RATE_PROFILES, offsetof(controlRateConfig_t, throttle_vbat_limit_percent) },
```

Comment:
To me it is totally unclear what this means. 'percent of what?'

Reply:
I tried many names. What I would like to describe is the level of battery voltage compensation (but this is not a compensation since it works the opposite way)

Reply:
So this is not a percentage at all? It should not be named `percent` then.

Reply:
I'll try to find a better name. My english is limited so any suggestion is appreciated!

Reply:
I think the problem here goes deeper than naming:

The value that you are configuring is very vague:
- it is dependent on `batteryConfig()->vbatwarningcellvoltage`, which is unintuitive and shouldn't be;
- the value returned by `calculateThrottleVbatLimitFactor()` will never even get close to 0 unless the craft is grossly misconfigured.
 Apart from the inherent problem that battery sag caused by this mechanism will lead to a feedback loop resulting in maximum battery power being applied every time the throttle is increase

Reply:
That factor is not intended to be close to zero! Again, why it should be?!

This observation make me think you didn't understand the sense of this feature.

I'm using the battery warning voltage as reference is because I want the throttle response to be stable **only** until the voltage reaches the warning level the pilot configured.

I read your suggestions for making this solution more complicated, but I ask you: did you tried it? You are used to write complex code based only on feelings?

Reply:
"resulting in maximum battery power being applied every time the throttle is increased" no, this feature instead avoids the use of maximum battery power. The multiplier is always less than one, so the output can only be reduced.